snap/snapcraft.yaml

```diff
@@ -1,6 +1,6 @@
 name: eclipse
 version: 'R'
-version-script: echo "4.8.0"
```

Comment:
I'd remove version-script completely, and just set version "2018-09" and that should fix the build failure.